src/components/learner-credit-management/data/hooks/useEnterpriseGroup.js

```diff
@@ -12,7 +12,7 @@ import LmsApiService from '../../../../data/services/LmsApiService';
  * @returns The enterprise group object
  */
 const getEnterpriseGroup = async ({ subsidyAccessPolicy }) => {
-  if (!subsidyAccessPolicy.groupAssociations || isEmpty(subsidyAccessPolicy.groupAssociations)) {
+  if (subsidyAccessPolicy === undefined || isEmpty(subsidyAccessPolicy.groupAssociations)) {
```

Comment:
nit
```suggestion
  if (isEmpty(subsidyAccessPolicy?.groupAssociations)) {
```

Reply:
If the policy didn't have any group associations, with the ? it would return undefined, and isEmpty(undefined) == false https://github.com/ramda/ramda/issues/2507

Reply:
We're using [lodash's isEmpty](https://lodash.com/docs/4.17.15#isEmpty) here, and when I tried it out on the lodash site, passing in `undefined` resulted in true.